pyquil/api/_qam.py

```diff
@@ -156,7 +156,7 @@ def read_from_memory_region(self, *, region_name: str) -> np.ndarray:
             DeprecationWarning,
         )
 
-        return self.read_memory(region_name=region_name)
+        return self.read_memory(region_name=region_name)  # type: ignore
```

Comment:
This is surprising. Looks like an easy one for mypy.

Reply:
I agree! I don't know why mypy couldn't figure this one out. Even trying to add `# type: ...` with the correct type name was not working.